app/workers/education_form/create_daily_year_to_date_report.rb

```diff
@@ -11,8 +11,8 @@ class CreateDailyYearToDateReport
       daily_processed: 0
     }.freeze
 
-    FORM_TYPE_HEADERS = EducationBenefitsClaim::FORM_TYPES.map do |form_type|
-      ["22-#{form_type}", '', '']
+    FORM_TYPE_HEADERS = EducationBenefitsClaim::FORM_HEADERS.map do |form_header|
+      [form_header.to_s, '', '']
```

Comment:
If these are the only two places you use `FORM_TYPE_HEADERS` you can still hardcode _the whole thing_

`'22-1990', '', '', '22-1995', , '', ''...` and avoid any need for someone to figure out that the headers actually look like.